apps/schools/api_views/school.py

```diff
@@ -117,19 +117,20 @@ class SchoolsInfo(SchoolsList, CacheMixin):
     def get_queryset(self):
         # inherit all the filtering from SchoolsList
         qset = super(SchoolsInfo, self).get_queryset()
-
+       
         qset = qset.select_related(
             'address', 'schooldetails__admin3',
             'schooldetails__admin2', 'schooldetails__admin1',
-            'schooldetails__type', 'schooldetails__assembly',
-            'schooldetails__parliament', 'electedrep__ward',
+            'schooldetails__type', #'schooldetails__assembly',
+            # 'schooldetails__parliament', 'electedrep__ward',
```

Comment:
Not sure if this is a bug in Django, but if you specify fields within `select_related` that don't exist, then the queryset itself becomes empty. So `'schooldetails__assembly', 'schooldetails__parliament', 'electedrep__ward',` have been commented out since they don't exist for Odisha right now.